### ISSUE TITLE
Certificates - app crash

### ISSUE THREAD

**Issue by mrfelek** [lensapp/lens#47]
**Describe the bug**
Entering to Certificates tab generates error.

**To Reproduce**
1. In Lens menu click Configuration
2. Click on 'Certificates'
3. See error

**Expected behavior**
Certificate list appears

**Screenshots**
![image](https://user-images.githubusercontent.com/17274927/66567223-0801b800-eb67-11e9-87fb-7cfef2a4eb1a.png)

**Environment (please complete the following information):**
- Lens Version: 2.1.2
- OS: Windows 10
- Installation method (e.g. snap or AppImage in Linux): Downloaded, updated package 
 
**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
 info: Checking for update
Loading kubeconfig from store with key: kubernetes-admin@kubernetes
Loading kubeconfig from store with key: aws
info: Update for version 2.1.2 is not available (latest version: 2.1.2, downgrade is disallowed).
(node:14608) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
(node:14608) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
info: Set kubectl version 1.16.1 for cluster version v1.15.3 using version map
info: initializing server for 10.0.0.31:6443 on port 9001
```

**Component stack:**
```
    in ItemListLayout
    in withRouter(ItemListLayout)
    in KubeObjectListLayout
    in withRouter(KubeObjectListLayout)
    in Certificates
    in Route
    in Switch
    in ErrorBoundary
    in main
    in div
    in MainLayout
    in withRouter(MainLayout)
    in Config
    in Route
    in Switch
    in Switch
    in ErrorBoundary
    in Router
    in b
    in App
```

**Error stack:**
```
TypeError: Cannot read property 'map' of undefined
    at Certificate.getConditions (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:2268604)
    at renderTableContents (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:2274894)
    at http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:1964232
    at Array.map (<anonymous>)
    at ItemListLayout.renderList (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:1963920)
    at ItemListLayout.render (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:1964947)
    at allowStateChanges (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:24218)
    at http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:77995
    at trackDerivedFunction (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:31388)
    at Reaction.track (http://d23c6a2d-5ad8-424f-9c5a-0b1858582a66.localhost:9000/app.js?9b761ca2cabeed8704f5:2:36439)
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: XXX
    server: https://10.0.0.31:6443
  name: kubernetes
contexts:
- context:
    cluster: kubernetes
    user: kubernetes-admin
  name: kubernetes-admin@kubernetes
current-context: kubernetes-admin@kubernetes
kind: Config
preferences: {}
users:
- name: kubernetes-admin
  user:
    client-certificate-data: XXX
    client-key-data: XXX
```

**Additional context**
I have second cluster configured (EKS) and it works there.

Kubernetes cluster on 
```
 cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

```

**Comment by jakolehm** [comment]
Fix included in v2.1.4.

**Comment by mrfelek** [comment]
Works.
Thank you!
